cluster-autoscaler/cloudprovider/huaweicloud/huawei-cloud-sdk-go/auth/aksk/aksk_options.go

```diff
@@ -0,0 +1,29 @@
+package aksk
```

Comment:
em.. Do you need to check in `huawei-cloud-sdk-go`? last branch already support go mod, why not use go mod to manage dependencies?

Reply:
Actually we encourage that pattern to keep core CA dependencies free of provider-specific ones. The idea is to limit top-level dependencies to core ones, so it's easy to fork and build a CA version for just one provider.
Keeping provider specific dependencies out is one of major motivations discussed in proposals like https://github.com/kubernetes/autoscaler/pull/3127. 

go.mod-extra is mostly meant for bumping version of clients that are already a transitive dependency of kubernetes/kubernetes.

Reply:
Thank you for the comments @Jeffwan @MaciekPytel .

For huawei-cloud-sdk-go, it's not the whole sdk published on "github.com/Huawei/cloud-sdk-go". Only the files that are needed for CA are put in there. Besides, the files that've been used are not the exact same code (some modifications have been made especially for CA) as the original sdk. In this way, even if the code on "github.com/Huawei/cloud-sdk-go" has changed, it has no impact on the CA part.